contrib/statistics-tex.py

```diff
@@ -58,25 +65,33 @@ class StatAccumulator(object):
     def __init__(self):
         self.count = 0
         self.cputime_values = []
+        self.walltime_values = []
 
     def add(self, result):
         self.count += 1
         self.cputime_values.append(extract_cputime(result))
+        self.walltime_values.append(extract_walltime(result))
 
     def to_latex(self, name_parts):
         cputime_stats = tablegenerator.StatValue.from_list(self.cputime_values)
+        walltime_stats = tablegenerator.StatValue.from_list(self.walltime_values)
         assert len(name_parts) <= 4
         name_parts += [""] * (4 - len(name_parts)) # ensure length 4
         name = r"}{".join(map(format_command_part, name_parts))
-        return "\n".join([
-            r"\StoreBenchExecResult{%s}{Count}{}{%s}" % (name, self.count),
-            r"\StoreBenchExecResult{%s}{Cputime}{}{%s}" % (name, cputime_stats.sum),
-            r"\StoreBenchExecResult{%s}{Cputime}{Avg}{%s}" % (name, cputime_stats.avg),
-            r"\StoreBenchExecResult{%s}{Cputime}{Median}{%s}" % (name, cputime_stats.median),
-            r"\StoreBenchExecResult{%s}{Cputime}{Min}{%s}" % (name, cputime_stats.min),
-            r"\StoreBenchExecResult{%s}{Cputime}{Max}{%s}" % (name, cputime_stats.max),
-            r"\StoreBenchExecResult{%s}{Cputime}{Stdev}{%s}" % (name, cputime_stats.stdev)
-            ])
+        return "\n".join(itertools.chain.from_iterable(
+            ["\StoreBenchExecResult{%s}{Count}{}{%s}" % (name, self.count)]
+          + [
+                r"\StoreBenchExecResult{%s}{%s}{}{%s}" % (name, time_name, time_stats.sum),
+                r"\StoreBenchExecResult{%s}{%s}{Avg}{%s}" % (name, time_name, time_stats.avg),
+                r"\StoreBenchExecResult{%s}{%s}{Median}{%s}" % (name, time_name, time_stats.median),
+                r"\StoreBenchExecResult{%s}{%s}{Min}{%s}" % (name, time_name, time_stats.min),
+                r"\StoreBenchExecResult{%s}{%s}{Max}{%s}" % (name, time_name, time_stats.max),
+                r"\StoreBenchExecResult{%s}{%s}{Stdev}{%s}" % (name, time_name, time_stats.stdev)
+              ] for (time_name, time_stats) in [
```

Comment:
Are you sure this works correctly? I would suspect that the `+` has a higher binding priority than the `for`, and thus the `Count` value appears once for each time.